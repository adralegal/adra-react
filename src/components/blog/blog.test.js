import React from 'react';
import userEvent from "@testing-library/user-event";
import renderer from "react-test-renderer";
import Blog from "./blog.tsx";



test ("Displays read more when blog is hovered", () => {
    const component = renderer.create(<Blog title="Hello" content="Hello" />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})