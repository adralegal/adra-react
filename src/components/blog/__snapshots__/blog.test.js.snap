// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays read more when blog is hovered 1`] = `
<div
  className="Wrapper"
>
  <h1>
    "
  </h1>
  <h3>
    Hello
  </h3>
  <p>
    Hello
  </p>
  <span>
    Read More...
  </span>
</div>
`;
